services/movie_session.py

```diff
@@ -42,3 +42,11 @@ def update_movie_session(
 
 def delete_movie_session_by_id(session_id: int) -> None:
     MovieSession.objects.get(id=session_id).delete()
+
+
+def get_taken_seats(movie_session_id: int) -> list[dict] | str:
+    tickets = Ticket.objects.filter(movie_session_id=movie_session_id)
```

Comment:
Not a mistake, but why not call function `get_movie_session_by_id` defined above?